### ISSUE TITLE
Jagurs in clustered environment

### ISSUE THREAD

**Issue by guntutur** [jagurs-admin/jagurs#7]
Hello there, a quick question : does jagurs support running in a multi machine environment like a server cluster? 
Say i have one master node with jagurs installed and configured, and 5 child node, all in one rack server on the same network. Is it possible with jagurs to utilise these 5 child node to fasten the computational time and process?

Best regards

**Comment by jagurs-admin** [comment]
Yes, JAGURS can use MPI for parallel running on a cluster. Please compile JAUGURS with a compile option of MPI=ON.
Thank you for having an interest in JAGURS.

**Comment by guntutur** [comment]
Could you point me to the documentation on how to set up and installation for clustered environment? thanks

**Comment by jagurs-admin** [comment]
"README" file in src directory explains the installation. For JAGURS compilation,  you need pre-installed libraries of NetCDF, NetCDF-fortran, HDF5, fftw, and proj4. And then you modify being correct paths to the libraries in Makefile and compile using MPI compiler (such as ifort) .  If you are not familiar with parallel computing, we recommend asking your system administrator for help. Some of the libraries may be already installed in your system.

**Comment by guntutur** [comment]
do you have any sample command to run jagurs admin within the cluster?
currently i have installed all needed libraries, including mpi and jagurs on master node
i have successfully run jagurs utilising openmpi on masternode but got stuck in utilising all the child nodes with mpi. thanks

**Comment by jagurs-admin** [comment]
Hi,
You can access to the sample data at
https://www.dropbox.com/sh/oiypdwufu31za1g/AADQ2QBRnu5BShO4Gh4989S4a?dl=0
I think that the parallel jagurs can run simply with
mpirun -np # jagrus par=tsun.par
Regards,

--
Baba, 088-656-9721 (ext. 4231)
